packages/form-js-viewer/src/render/components/form-fields/Checkbox.js

```diff
@@ -37,22 +38,26 @@ export default function Checkbox(props) {
 
   const { formId } = useContext(FormContext);
 
-  return <div class={ formFieldClasses(type, errors) }>
-    <Label
-      id={ prefixId(id, formId) }
-      label={ label }
-      required={ false }>
-      <input
-        checked={ value }
-        class="fjs-input"
-        disabled={ disabled }
-        id={ prefixId(id, formId) }
-        type="checkbox"
-        onChange={ onChange } />
-    </Label>
-    <Description description={ description } />
-    <Errors errors={ errors } />
-  </div>;
+  return (
+    <div
+      class={ classNames(formFieldClasses(type, { errors, disabled }), {
+        'fjs-checked': value,
```

Comment:
Should we move this to `formFieldClasses` as well or do you think it's too specific?

Reply:
I think it makes sense to leave it localized since it's not relevant to all components